File: app/src/main/java/com/ravikoradiya/zoomableimageviewmaster/MainActivity.java
package com.ravikoradiya.zoomableimageviewmaster;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.ravikoradiya.zoomableimageview.ZoomableImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ZoomableImageView zoomableImageView = (ZoomableImageView) findViewById(R.id.iv_zoomable);
        zoomableImageView.setImageUrl("http://...");
    }
}

File: app/src/main/java/com/ravikoradiya/zoomableimageviewmaster/MainActivity.kt
package com.ravikoradiya.zoomableimageviewmaster

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.ravikoradiya.zoomableimageview.ZoomableImageView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val zoomableImageView = findViewById(R.id.iv_zoomable) as ZoomableImageView
        zoomableImageView.setImageUrl("http://...")


    }
}
